kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/CustomResourceCrudTest.java

```diff
@@ -108,9 +107,11 @@ void testCreateOrReplaceRaw() throws IOException {
 
     object.put("spec", "updated");
 
-    Map<String, Object> updated = raw.createOrReplace(object);
-    assertNotEquals(created, updated);
-    assertEquals(object, updated);
+    // with the mock detecting conflicts, this won't work because the
+    // appropriate attributes are extracted to perform the get after the 409
+    //Map<String, Object> updated = raw.createOrReplace(object);
+    //assertNotEquals(created, updated);
+    //assertEquals(object, updated);
```

Comment:
Does this mean createOrReplace will no longer work in crud mode?

Reply:
More broadly it means that GET doesn't work in raw mode - the metadata is not extracted properly during create.

Reply:
I've updated this pr with a fix to that problem and another change to not send update events when nothing changes.